src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/StructureFormMetadataLoader.php

```diff
@@ -121,6 +129,11 @@ public function warmUp($cacheDir)
         foreach ($this->locales as $locale) {
             foreach ($structuresMetadataByTypes as $structureType => $structuresMetadata) {
                 $structure = $this->mapStructureMetadata($structuresMetadata, $locale);
+
+                foreach ($structure->getForms() as $formMetadata) {
+                    $this->validateFormMetadata($formMetadata);
```

Comment:
I have no idea how to test this, because this is in the `warmUp()` method. Calling this method in a test doesn't throw an exception